src/cloudWatchLogs/commands/copyLogStreamName.ts

```diff
@@ -20,7 +20,13 @@ export async function copyLogStreamName(uri?: vscode.Uri): Promise<void> {
             }
         }
         const parsedUri = parseCloudWatchLogsUri(uri)
-        await vscode.env.clipboard.writeText(parsedUri.streamName)
+        const logGroupInfo = parsedUri.logGroupInfo
+
+        if (!logGroupInfo.streamName) {
+            throw new Error()
+        }
+
+        await vscode.env.clipboard.writeText(logGroupInfo.streamName)
```

Comment:
is the messages.copyToClipboard() function useful here?

Reply:
it's just a small bit of cleanup while we're here. `messages.copyToClipboard` is the common util we want to use everywhere when we do a "copy to clipboard" action.